docx/document.py

```diff
@@ -27,6 +27,15 @@ def __init__(self, element, part):
         super(Document, self).__init__(element)
         self._part = part
         self.__body = None
+        
+    def add_bookmark_start(self, name):
+        paragraph = self.add_paragraph()
+        return paragraph.add_bookmark_start(id, name)
+
+    def add_bookmark_end(self, name):
+        id = 1
+        paragraph = self.add_paragraph()
+        return paragraph.add_bookmark_end(id)
```

Comment:
I don't think most of the rest of these changes are quite right, but let's wait until you get these organized into more final form before I invest time in a critique.

I will say though that the client interface is not quite right here. In general we don't pass/return a raw element class to a client; this would be a `Bookmark` object instead (or some other interface object like that). I assume you did it this way just to get something working and maybe not have to create a `Bookmark` class quite yet. We can get into that once we get further along.